src/lib/ui/PasswordStrengthIndicator.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, ReactNode } from 'react';
+
+import { T } from '../i18n/react';
+
+export interface PasswordValidation {
+  minChar: boolean;
+  cases: boolean;
+  number: boolean;
+  specialChar: boolean;
+}
+
+type PasswordStrengthIndicatorProps = {
+  validity: PasswordValidation;
```

Comment:
Try not to create a lot of the same spelling words. Mb `validation: PasswordValidation`?